src/NuGet.Core/NuGet.Packaging/Licenses/NuGetLicenseLogicalOperatorType.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Packaging
+{
+    /// <summary>
+    /// Represents the logical operator type of a <see cref="NuGetLicenseLogicalOperator"/>.
+    /// </summary>
+    public enum NuGetLicenseLogicalOperatorType
+    {
+        AND,
```

Comment:
>AND [](start = 8, length = 3)

nit: This casing is inconsistent with the rest of the enums AFAIK. `And` and `Or` would be more consistent. Note that if you go this route the `ToString` of `LicenseOperator` will look weird so maybe fix that up too if you make this change.

Reply:
Yeah get your point. 
Fixed. 
I'd rather have the types be consistent.